task_sender/cmd/main.go

```diff
@@ -159,6 +158,8 @@ func parseProvingSystem(provingSystemStr string) (common.ProvingSystemId, error)
 	switch provingSystemStr {
 	case "plonk":
 		return common.GnarkPlonkBls12_381, nil
```

Comment:
to be consistent in the namings,
```suggestion
	case "plonk_bls12_381":
		return common.GnarkPlonkBls12_381, nil
```